Felienne/hedy
# [FEATURE] Localized quiz keywords

**Description**
In this PR we localize the quiz keywords to align with our implementation of localization for the level-defaults and adventures. Due to the current structure of the quizzes this had to be re-written as well. We now retrieve quiz data similar to level-defaults and adventures by reading them once on setup and then reading from the created dict. This should be more efficient than the current implementation where we read the YAML file again and again (each question!).

**Fixes**
This PR fixes #2103 (but got a bit out-of-hand and fixes the quiz structure as well)

**How to test**
Verify that the quiz is now working for every language. If no quiz template exists the English one is automatically returned. (We might want to do this for level defaults as well?) Keywords are automatically parsed from the selected keyword language, if non is selected these are English. Verify that this is working correct as well.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
